integration-tests/src/test/java/org/zowe/apiml/functional/gateway/GatewayAuthenticationTest.java

```diff
@@ -62,14 +67,15 @@ void thenAuthenticate() {
     class GivenInvalidBearerAuthentication {
         @Nested
         class WhenAccessingProtectedEndpoint {
-            @Test
-            void thenReturnUnauthorized() {
+            @ParameterizedTest
+            @ValueSource(strings = {"/application", "/application/health"})
```

Comment:
Should really `/application` returns 401?

Reply:
yes, https://github.com/zowe/api-layer/blob/6e301af36477a6fb7fc1a0370e7079fbcb246d7d/zaas-service/src/main/java/org/zowe/apiml/zaas/security/config/NewSecurityConfiguration.java#L492